pkg/virt-launcher/virtwrap/converter/vcpu/vcpu.go

```diff
@@ -0,0 +1,282 @@
+package vcpu
+
+import (
+	"fmt"
+	"strconv"
+
+	v1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+type VCPUPool interface {
+	FitCores() (tune *api.CPUTune, err error)
+	FitThread() (thread uint32, err error)
+}
+
+func CalculateRequestedVCPUs(cpuTopology *api.CPUTopology) uint32 {
+	return cpuTopology.Cores * cpuTopology.Sockets * cpuTopology.Threads
+}
+
+type cell struct {
+	bigThreadJunks [][]uint32
+	smallJunks     []uint32
+	threadsPerCore int
+}
+
+// GetCPUBoundJunk consumes the amount of threadsPerCore from the numa cell
+// or none if it can't be fit on the numa cell
+func (c *cell) GetCPUBoundJunk() []uint32 {
```

Comment:
what does the term `junk` mean in relation to all this cpu mapping?

Reply:
As @kwiesmueller mentioned I meant `chunk` :D

Reply:
Renamed.